github-markdown.css

```diff
@@ -677,6 +757,15 @@
   border-color: #0366d6;
 }
 
+.markdown-body .new-label-color-input:invalid {
+  color: #cb2431;
+}
```

Comment:
I don't think this one is relevant to markdown.

Reply:
Agreed, but for some reason it's getting through `uncss`.  I'll poke around a bit and see if I can figure out why, and how to fix it without a manual override.

Reply:
Updating `uncss` got rid of this.